source/sundials/ida.cc

```diff
@@ -151,11 +159,79 @@ namespace SUNDIALS
 
       return err;
     }
+#  else
+
+
+
+    template <typename VectorType>
+    int
+    t_dae_jacobian_setup(realtype tt,
+                         realtype cj,
+                         N_Vector yy,
+                         N_Vector yp,
+                         N_Vector /* ignored */,
+                         SUNMatrix /* ignored */,
+                         void *user_data,
+                         N_Vector /* ignored */,
+                         N_Vector /* ignored */,
+                         N_Vector /* ignored */)
```

Comment:
Out of curiosity, what are all of these arguments? In the case of KINSOL, I ended up adding a new function object that actually makes use of these vectors because they turned out to be useful.

Reply:
These are actually three tmp vectors that IDA provides in case they are needed. For the time being they are not, and I have no idea what they could be, but I stated what they are in the comment.